app/routes.py

```diff
@@ -1,2 +1,97 @@
-from flask import Blueprint
+from unicodedata import name
+from flask import Blueprint, jsonify, abort, make_response, request
+from app import db
+from app.models.planet import Planet
 
+# class Planet:
+
+#     def __init__(self, id, name, description, moons):
+#         self.id = id
+#         self.name = name
+#         self.description = description
+#         self.moons = moons
+
+# planets = [
+#     Planet(1, "Earth", "our home", 1),
+#     Planet(2, "Mars", "red planet", 2),
+#     Planet(3, "Pluto", "is a planet", 5)
+# ]
+
+planets_bp = Blueprint("planets", __name__, url_prefix="/planets")
+
+@planets_bp.route("", methods=["POST"])
+def create_planet():
+    request_body = request.get_json()
+    if ("name" not in request_body or "description" not in request_body
+    or "moons" not in request_body):
+        return make_response(f"Invalid Request", 400)
```

Comment:
Great validation, could this be abstracted in to a helper method and used in the update action as well?